Spring-Boot/Spring-Boot-Submission/src/test/java/tw/waterball/judgegirl/springboot/submission/controllers/AbstractSubmissionControllerTest.java

```diff
@@ -366,4 +362,14 @@ protected ResultActions requestWithToken(Supplier<MockHttpServletRequestBuilder>
                 .header("Authorization", "bearer " + token));
     }
 
+    protected VerdictView createVerdictAndGet(SubmissionView actualSubmission,
+                                              Submission submission) {
+        submission.setProblemId(actualSubmission.problemId);
+        submission.setStudentId(actualSubmission.studentId);
+        ReportView report = ReportView.toViewModel(ProblemStubs.compositeReport());
+        VerdictView verdict = VerdictView.toViewModel(submission.getVerdict().orElseThrow());
+        verdict.setReport(report);
+        return verdict;
+    }
```

Comment:
Delete this method, instead, we use `VerdictStubBuilder` (may create it yourself)

delegate the verdict's construction process from `SubmissionStubBuilder` to `VerdictStubBuilder`